Rubjerg.Graphviz/Graph.cs

```diff
@@ -483,14 +483,27 @@ public void FreeLayout()
                 throw new ApplicationException($"Graphviz render returned error code {free_rc}");
         }
 
+        private void RenderToFile(string filename, string format)

```

Comment:
This is the original code which was moved to a private method to reuse it from both callers.